apps/safe-claiming-app/src/components/steps/SafeInfo/index.tsx

```diff
@@ -1,16 +1,28 @@
-import { Card, Paper, Typography } from "@mui/material"
+import { Card, Paper, styled, Typography } from "@mui/material"
 import Box from "@mui/system/Box"
 
 import { ReactComponent as LockIcon } from "src/assets/images/lock.svg"
 import { ReactComponent as ValueIcon } from "src/assets/images/assets.svg"
 import { NavButtons } from "src/components/helpers/NavButtons"
+import { useDarkMode } from "src/hooks/useDarkMode"
 
 type Props = {
   handleBack: () => void
   handleNext: () => void
 }
 
+const InfoCard = styled(Card)<{ darkMode: boolean }>(({ darkMode, theme }) => ({
+  backgroundColor: darkMode
+    ? theme.palette.background.paper
+    : theme.palette.safeGreen.light,
```

Comment:
I would suggest using `safeGreen.light` for both dark and light mode.

Reply:
I put `safeGreen.light` for both in a first version, but lila explicitly wanted it to be colored like this. 
She marked everything which was important for her with these red boxes.

https://www.figma.com/file/igarLtnMhVtrYNOATmjJDc/Claim-S-Tokens-App?node-id=2363%3A15272&t=OB4Zd1XArFbKL8p3-0